cmd/PolicyGenerator/main.go

```diff
@@ -19,9 +19,16 @@ var debug = false
 func main() {
 	// Parse command input
 	debugFlag := pflag.Bool("debug", false, "Print the stack trace with error messages")
+	helpFlag := pflag.BoolP("help", "h", false, "Print the help message")
 	versionFlag := pflag.Bool("version", false, "Print the version of the generator")
 	pflag.Parse()
 
+	if *helpFlag {
+		fmt.Println("Usage: PolicyGenerator [flags] <policy-generator-config-file>...")
```

Comment:
Looks like the linter doesn't like the `fmt.Println`, but it's acceptable here.

```suggestion
		//nolint:forbidigo
		fmt.Println("Usage: PolicyGenerator [flags] <policy-generator-config-file>...")
```